.codeclimate.yml

```diff
@@ -0,0 +1,5 @@
+exclude_patterns:
+  - "**/*_test.go"
+  - "**/test/"
+  - "**/tests/"
```

Comment:
not sure about those entries

Reply:
Without `**/test/` code climate was complaining about multiple files not related to this PR.

Reply:
And it's included in the default config, ok.